src/core/model_config.proto

```diff
@@ -764,6 +766,30 @@ message ModelParameter
   string string_value = 1;
 }
 
+//@@
+//@@.. cpp:var:: message ModelWarmUp
+//@@
+//@@   Settings for warming up the model instances.
+//@@
+message ModelWarmUp
+{
+  //@@  .. cpp:var:: InferRequestHeader request_header
```

Comment:
Give a name... will be useful for logging and I think it will also be necessary for the "actual data" case.